apple/internal/apple_framework_import.bzl

```diff
@@ -123,6 +126,12 @@ def _apple_framework_import_impl(ctx):
     dummy_binary = ctx.actions.declare_file("_{}.dummy_binary".format(ctx.label.name))
     ctx.actions.write(dummy_binary, "_dummy_file_")
 
+    objc_provider_fields["providers"] = [
+        dep[apple_common.Objc]
+        for dep in ctx.attr.deps
+        if apple_common.Objc in dep
```

Comment:
I guess I could remove this since the deps field requires it with its providers list